### ISSUE TITLE
Tag the JavaScript properly

### ISSUE THREAD

**Issue by fmarier** [mozilla/srihash.org#20]
Once we start using client-side JavaScript, we should license it properly so that it doesn't get blocked by [LibreJS](https://www.gnu.org/software/librejs/).

Instructions on how to do that are here:
- https://www.gnu.org/software/librejs/manual/librejs.html#Setting-Your-JavaScript-Free
- https://www.gnu.org/software/librejs/free-your-javascript.html#step3
- https://www.gnu.org/licenses/javascript-labels.html

**Comment by mozfreddyb** [comment]
I'll dare considering this out of scope given the time I want and can allocate. But pull requests very welcome :)
